_config.yml

```diff
@@ -21,11 +21,6 @@ youtube_channel: UC-rLOtGfHGx9N1mlNNEwJNQ
 discord: https://discord.gg/3xxec7V5zN
 remote_theme: pages-themes/hacker@v0.2.0
 
-sponsors:
```

Comment:
Are we not showing sponsors anymore? Also, if we are, you should also delete the actual file that this was linked to before

Reply:
Oh wait, I see this is just the config. Disregard that.

Reply:
Yeah this is used to auto populate the sponsors at the bottom of each page, and it was doubled up for some reason? The site did not appear to change when I removed it, I should have probably added that into commit comment too.